### DISCUSSION TITLE
Change device type?

### DISCUSSION THREAD

**Discussion by YellowNest** [Koenkk/zigbee2mqtt#9201] [Devices]
I have a couple of IKEA Trådfri control outlets.
In Homekit they are displayed as switches.
I wonder if it's possible to change them to wall outlets instead?
Is it possible to just add something to configuration.yaml to make this happen?

**Comment by Koenkk** [comment]
You should create this issue in the zigbee2mqtt <-> homekit bridge software, zigbee2mqtt doesnt directly talk to homekit.

**Reply by YellowNest** [reply]
Thanks, I hope @itavero will fix it one day then
